src/exceptions.py

```diff
@@ -66,3 +66,35 @@ def __init__(self, msg: str):
             msg (str): Explanation of the error.
         """
         self.msg = msg
+
+
+class PeerRelationUnavailableError(Exception):
+    """Exception raised when the peer relation is unavailable.
+
+    Attrs:
```

Comment:
I think to get the `src-docs` working this would need to be renamed to `Attributes` for now unfortunately.

Reply:
Fixed